packages/react-native-web/src/exports/ScrollView/ScrollViewBase.js

```diff
@@ -137,12 +137,8 @@ export default class ScrollViewBase extends Component<*> {
   _createPreventableScrollHandler = (handler: Function) => {
     return (e: Object) => {
       if (this.props.scrollEnabled) {
-        if (handler) {
+        if (handler)
           handler(e);
-        }
-      } else {
-        // To disable scrolling in all browsers except Chrome
-        e.preventDefault();
```

Comment:
Why is this code removed?

Reply:
To be clearer, when the scrolling is disabled:
1. `touchAction: 'none'` doesn't prevent scrolling
2. `e.preventDefault()` was the real code which was preventing the scrolling. The problem is that this function does too much, *it catches the event, and stop it*. This prevent *all scrolling*, even for scrolling with outside views of the `ScrollView` (parents view) because the event is catched.

Reply:
As the comments explain, different browsers required different ways to disable scrolling without removing the scroll bars or changing the scroll position. Propagation of the scroll event up to ancestors is a problem

Reply:
The comment on `preventDefault` says "To disable scrolling in all browsers except Chrome".
But it does disable scroll in chrome too. And, as you said, it stops the propagation of the scroll event up to ancestors. Then that really doesn't seem a good way to prevent scroll.

The new way to disable scroll using style's `overflowX` & `overflowY` properties works for all browser. You can test it in this sandbox with the browser of your choice.
Test on browser: https://w6yo77mrp5.codesandbox.io
Code: https://w6yo77mrp5.codesandbox.io/s/w6yo77mrp5

Reply:
This patch doesn't seem to fix the problem either. Using your test case + this patch, when a touch-scroll starts over the disabled horizontal `FlatList` it doesn't scroll the parent `ScrollView`.

Reply:
Yes you're right about the **touch** -scroll.
But the ` e.preventDefault()` won't help you anyway. ^^

Reply:
> This patch doesn't seem to fix the problem either. Using your test case + this patch, when a touch-scroll starts over the disabled horizontal `FlatList` it doesn't scroll the parent `ScrollView`.

Is this the problem related to #1160 ?